docs/book/v1/intro.md

```diff
@@ -42,6 +40,37 @@ if a user has been authenticated or not, e.g. it can be used to verify the
 authorization level of a user (for this scope, it is consumed by
 [zend-expressive-authorization](https://github.com/zendframework/zend-expressive-authorization)).
 
+## Default User class
+
+We provide a default implementation of `UserInterface` via the class `Zend\Expressive\Authentication\DefaultUser`.
+This class is a basic implementation of `UserInterface`. This class is final and
+it's immutable, that means you cannot change it state at runtime.
```

Comment:
Remove the sentence "This class is a basic implementation..."; it's redundant.

Rephrase the next sentence to: "The class is final and immutable, in order to prevent runtime changes."

Reply:
Ok